goldenmaster/Plugins/ApiGear/Source/ApiGearOLink/Private/OLinkHostConnection.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+THIRD_PARTY_INCLUDES_START
+#include "olink/remoteregistry.h"
+#include "olink/remotenode.h"
+THIRD_PARTY_INCLUDES_END
+#include <UObject/Object.h>
+#include "ApiGearLogCategories.h"
+#include "INetworkingWebSocket.h"
+
+/// @brief an instance of this class is held by the OLink server worker for each existing connection
```

Comment:
an instance of this class is held by the OLink server worker for each existing connection
-> handles exclusively a connection operated by one WebSocket.
Binds data from WebSocket to a RemoteNode which is an OLink network endpoint abstraction

Reply:
done